deploy/crds/installation.crd.yaml

```diff
@@ -35,8 +35,18 @@ spec:
               enum:
               - workshop
               - managed
+            namespacePrefix:
+              type: string
+            selfSignedCerts:
+              type: boolean
+            githubOauthClientId:
+              type: string
+            githubOauthClientSecret:
+              type: string
           required:
             - type
+            - githubOauthClientId
```

Comment:
These GitHub credentials aren't currently required in the installer. We could follow the same pattern of allowing for placeholder values until the CR is updated to be more flexible? Allowing for the GitHub OAuth app to be created after the cluster has been provisioned.

Reply:
Agree with the placeholder, the github oauth secret should be updated on reconcile. I think having a placeholder here would work and that would prevent the Launcher console giving an error due to missing oauth client id and secret